semgrep_output_v0.atd

```diff
@@ -229,7 +229,9 @@ type core_error
 type core_error_kind
   <ocaml attr="deriving show">
   <python decorator="dataclass(frozen=True, order=True)"> = [
-  (* File parsing related errors *)
+  (* File parsing related errors; if you add a target parse error then
```

Comment:
I usually add a coupling: tag to the comment.
(* File parsing related errors.
 * coupling: if you do this ... you probably need to modify that ...
 *)